src/lib.rs

```diff
@@ -99,6 +99,14 @@ impl ZWaveManager {
         }
     }
 
+    pub fn add_node(&mut self, home_id: u32, secure: bool) -> Result<(), ()> {
```

Comment:
the lib uses mutexes for this call, so we don't need a &mut, a shared reference is good enough.